### PR TITLE
Set yaml default code point limit

### PR DESCRIPTION
**Pull request by usmansaleem**
## PR Description
Set yaml default code point limit.
 -- Default to 100MB
 -- Introduced cli option `--key-store-config-files-size`

## Fixed Issue(s)

#719 
## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+import java.util.Optional;
+
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import org.yaml.snakeyaml.LoaderOptions;
+
+public class YamlMapperProvider {
+  private static final int YAML_DEFAULT_CODE_POINT_LIMIT =
```
Comment (jframe) [inline signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java:25]: This is a really high limit. Maybe 100MB?

File: signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      1_073_741_824; // 1024 * 1024 * 1024 = 1GB
+  private static final String SYSTEM_PROPERTY = "web3signer.yamlCodePointLimit";
```
Comment (jframe) [inline signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java:27]: Seems like this should a pico cli option so we get all the ways to configure it like other options

Comment (jframe) [inline signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java:27]: The name probably won't mean anything to anyone who doesn't know snakeyaml.
If the limit the same of the max file size? Maybe maxYamlFileSize?

File: signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java (hunk 1 above)
Comment (usmansaleem) [reply signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java]: agreed, updated the code to use cli option instead.

File: signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java (hunk 2 above)
Comment (usmansaleem) [reply signing/src/main/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlMapperProvider.java]: agreed, updated the code to use cli option instead.

File: commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java [hunk 3]
```diff
@@ -82,11 +82,20 @@ public class Web3SignerBaseCommand implements Config, Runnable {
...
-      description = "The path to a directory storing toml files defining available keys",
+      description = "The path to a directory storing yaml files defining available keys",
       paramLabel = DefaultCommandValues.MANDATORY_PATH_FORMAT_HELP,
       arity = "1")
   private Path keyStorePath = Path.of("./");
 
+  @Option(
+      names = {"--key-store-config-files-size"},
+      description =
+          "The key store configuration file size in bytes. Useful when loading large number of "
```
Comment (jframe) [inline commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java:93]: Similarly we should mention that this is the max file size

File: commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java [hunk 4]
```diff
@@ -82,11 +82,20 @@ public class Web3SignerBaseCommand implements Config, Runnable {
...
   @Option(
       names = {"--key-store-path"},
-      description = "The path to a directory storing toml files defining available keys",
+      description = "The path to a directory storing yaml files defining available keys",
       paramLabel = DefaultCommandValues.MANDATORY_PATH_FORMAT_HELP,
       arity = "1")
   private Path keyStorePath = Path.of("./");
 
+  @Option(
+      names = {"--key-store-config-files-size"},
```
Comment (jframe) [inline commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java:91]: maybe --key-store-config-files-max-size?

File: commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java [hunk 5]
```diff
@@ -82,11 +82,20 @@ public class Web3SignerBaseCommand implements Config, Runnable {
...
+      description =
+          "The key store configuration file size in bytes. Useful when loading large number of "
+              + "configurations from same yaml file. Defaults to 100MB (104857600) ",
+      paramLabel = "<NUMBER>",
+      arity = "1")
+  private final Integer keystoreConfigFileSize = 104_857_600;
```
Comment (jframe) [inline commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java:97]: Is integer large enough considering this is bytes? maybe long would be better

Comment (usmansaleem) [reply commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerBaseCommand.java:97]: the yaml codePoint method takes an int.

**Comment by usmansaleem** [comment]
Close due to #729

**Pull request closed without merging**